manifests/database.package.yaml

```diff
@@ -1,5 +1,5 @@
-packageName: db-operators
+packageName: REPLACE_PACKAGE
 channels:
-- currentCSV: postgresql-operator.v0.0.3
+- currentCSV: REPLACE_NAME.vREPLACE_VERSION
```

Comment:
Would be nice to not have this `vREPLACE_VERSION` to only keep `REPLACE_VERSION`. Another approach would be adding `REPLACE_IMAGE_TAG` where you bring together the prefix you would need to reflect the image tag.

Reply:
@otaviof Interesting idea, but let's solve this via another PR, the current solution, while not being ideal, still works.